mmpose/datasets/pipelines/pose3d_transform.py

```diff
@@ -0,0 +1,265 @@
+import mmcv
+import numpy as np
+import torch
+from mmcv.utils import build_from_cfg
+
+from mmpose.core.camera import CAMERAS
+from mmpose.datasets.registry import PIPELINES
+
+
+@PIPELINES.register_module()
+class JointRelativization:
+    """Zero-center the pose around a given root joint. The root joint will be
+    removed from the original pose and optionally stored as a separate item.
+
+    Note that the relativized joints no longer align with some annotation
+    information (e.g. flip_pairs, num_joints, inference_channel, etal) due to
+    the removal of the root joint.
+
+    Args:
+        item (str): The name of the pose to relativeze.
+        root_index (int): Root joint index in the pose.
+        root_name (str): Optional. If not none, it will be used as the key to
+            store the root position separated from the original pose.
+
+    Required keys:
+        item (depend on args): Should be an array in shape [...,Kj,C], where
```

Comment:
Kj -> K

Reply:
Emmm. There is a problem. K is also the abbreviation of intrinsic matrix.

Reply:
We plan to use J for joint number in the docstring

Reply:
When to use K and when to use J?

Reply:
Only use J in the camera part.